build_tools/sdk.py

```diff
@@ -134,7 +134,7 @@ def _replace_pattern(s, patterns):
             s = s.replace(pattern, replace)
         return s
 
-    dirs.sort(key=lambda x: tuple(int(token) for token in _replace_pattern(x, replace_patterns).split('.')))
+    dirs.sort(key=lambda x: tuple(int(token) for token in _replace_pattern(x, replace_patterns).split('.')), reverse=True)
```

Comment:
Largest numbers first

Reply:
Side note and unrelated, but I really don't like python lambda functions. I really have to pause and look closely at the code to understand what it really does.... :-/